### ISSUE TITLE
Lottery export preference tabs should be in order of ordinal

### ISSUE THREAD

**Issue by ludtkemorgan** [metrotranscom/doorway#877]
When creating the lottery export the preference tabs should be ordered by the ordinal of the preference. Currently they are just ordered in the order they come back from the database.

The highest priority preference should be the second tab (the tab immediately after the raw ranking tab). and the second highest priority preference should be the third tab. etc

**Comment by jaredcwhite** [comment]
@ludtkemorgan @sarahlazarich It looks like the code where I'd be updating this is the same in Doorway and Core. Should I update in Core first and then bring this over?

**Comment by ludtkemorgan** [comment]
@jaredcwhite yeah, we should have this in core as well. Let me know if you still need help testing it
